### ISSUE TITLE
Section with all other features

### ISSUE THREAD

**Issue by lumio** [cheesein/graphics#4]
* Custom multiple check in states
* Touchable signation (hardware required)
* Status check on multiple devices
* People database
* Easy label printing
* Anonymous numbering
* Automatic groups (changeable)
* Manual groups

add graphic to each point under CTA button

Like http://electron.atom.io/ quick features

**Comment by DDCreationStudios** [comment]
## Graphic suggestions just for inspiration:
- traffic light for status 
![image](https://cloud.githubusercontent.com/assets/22077628/21753954/39c3bc5e-d5f7-11e6-99ee-4c29fa7d16aa.png)
- touch signature (add signature) 
![image](https://cloud.githubusercontent.com/assets/22077628/21753957/526ad896-d5f7-11e6-8ddb-bd4087030df6.png)
- multiple devices 
![image](https://cloud.githubusercontent.com/assets/22077628/21753947/0dda0f3a-d5f7-11e6-829c-7ac8ec4212cc.png)
- database 
![image](https://cloud.githubusercontent.com/assets/22077628/21753962/6f23459a-d5f7-11e6-8226-310b944b04f8.png)
- labeling
![image](https://cloud.githubusercontent.com/assets/22077628/21753981/bc3ec2aa-d5f7-11e6-816e-9a402f730286.png)
- anonymous
![image](https://cloud.githubusercontent.com/assets/22077628/21753984/ca06cf7c-d5f7-11e6-9c97-c25a9b413e37.png)
- groups
![image](https://cloud.githubusercontent.com/assets/22077628/21753996/e7f817ac-d5f7-11e6-87ee-00c56db55701.png)

**Comment by lumio** [comment]
Thanks! Where did you find those? On flaticon?
I guess this will help me a lot

**Comment by DDCreationStudios** [comment]
Yeah. But I also checked other sites ;) I think these are quite good for basic icons :)
